### ISSUE TITLE
Fix Deprecated ImageCropper API

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa#1203]
**Describe the bug**
When building the `talawa` app we get an ImageCropper API warning message. This needs to be fixed

- Our test code coverage percentage will need to be improved or maintained as part of this fix.
- **DO NOT ATTEMPT to solve the issue by ignoring the error message in `build.gradle`** That is not a viable solution to this issue.

**To Reproduce**

1. Build the Talawa app
2. Monitor the console
3. See message below

**Expected behavior**
There should be no deprecation warning

**Actual behavior**
```
Note: /home/xxx/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/image_cropper-1.4.1/android/src/main/java/vn/hunghd/flutter/plugins/imagecropper/ImageCropperPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
```

**Screenshots**
N/A

**Additional details**
N/A

**Comment by Tusharkhati** [comment]
I'm really sorry. I was supposed to `Fix Deprecated UniLinksPlugin API #1204`. I committed wrongly I will take care of this in the future.

**Comment by noman2002** [comment]
Please assign this to me. I am willing to work on it.
